### PR TITLE
[WIP] InterfaceManager: fix imports

### PR DESCRIPTION
**Pull request by svinota**
These were moved a long time ago. Used in pyroute2 from 0.2.15::

    # pyroute2.netlink.rtnl
    RTM_NEWLINK
    RTM_DELLINK
    RTM_NEWADDR
    RTM_DELADDR

These were *renamed* in the kernel. Used in pyroute2 from 0.5.1::

    # pyroute2.netlink.rtnl
    RTMGRP_IPV4_IFADDR
    RTMGRP_IPV6_IFADDR
    RTMGRP_LINK

The details about the multicast groups see the pyroute2 git log:
 * c435651eca552db32af15c552c1b9a3a5aa8e350
 * 39fa987f9d6876966dad5be1b911e54f597dc8b3

### CODE REVIEW

File: lnst/Slave/InterfaceManager.py [hunk 1]
```diff
@@ -24,24 +24,18 @@
...
-    from pyroute2.netlink.iproute import RTM_NEWADDR
-    from pyroute2.netlink.iproute import RTM_DELADDR
-except ImportError:
-    from pyroute2.iproute import RTM_NEWLINK
-    from pyroute2.iproute import RTM_DELLINK
-    from pyroute2.iproute import RTM_NEWADDR
-    from pyroute2.iproute import RTM_DELADDR
+from pyroute2.netlink.rtnl import RTMGRP_IPV4_IFADDR
+from pyroute2.netlink.rtnl import RTMGRP_IPV6_IFADDR
+from pyroute2.netlink.rtnl import RTMGRP_LINK
```
Comment (svinota) [inline lnst/Slave/InterfaceManager.py]: maybe split this into two different commits?

**Comment by olichtne** [comment]
I haven't forgotten about this :). now that we've ported the next branch to Python 3 I think it's also a good idea to update it to use newer  pyroute2.

At the same time I don't expect any important new stuff to be happening on the master branch so I'd say I'm okay with keeping it working with just the older pyroute2 version (up to 0.4.21).

So I'm going to try to change the base branch of this to the next branch and merge it there if it still applies.

**Comment by olichtne** [comment]
Never mind... I was afraid that it would attempt to pull in other changes as well.

Second option is for me to just manually cherry-pick this one commit into next and just close this pull request without merging.

**Comment by olichtne** [comment]
Commit cherry-picked to next, resolved a merge conflict and pushed here: 042f6246133824cb59eba7b4e457807051e6f1b1

I also added one more commit (f72e8483afdb3888ebd5f2cd191f67ca96a4b4fd) that does a similar fix in the lnst.Devices.Device module.

Thanks @svinota for the merge request and sorry that it took so long :).
